workloads/fio/fio.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+
+# Log file will be on pod
+log = logging.getLogger(__name__)
+
+
+def setup(**kwargs):
+    """
+    setup fio workload
+
+    Args:
+        **kwargs (dict): fio setup configuration
+
+    Returns:
+        bool: True if setup succeeds else False
+    """
+    io_pod = kwargs['pod']
+    # For first cut doing simple fio install
+    # TODO: we can use fio container image
```

Comment:
I would remove this TODO, I think there is not good advantage of using its own image, since we still have to configure it like worker pod, configuring it on the fly for any pod is needed.

Reply:
Done